packs/_source/monsters/beast/ape.json

```diff
@@ -132,7 +132,7 @@
     },
     "details": {
       "biography": {
-        "value": "",
+        "value": "<p><em>Token artwork by </em><a href=\"https://www.forgotten-adventures.net/\" target=\"_blank\" rel=\"noopener\"><em>Forgotten Adventures</em></a><em>.</em></p>",
```

Comment:
Shouldn't be baked in

```suggestion
        "value": "",
```

Reply:
oh, ok, that's makes more sense -- i was a bit confused when i saw that in the diff, but it seemed like a legit change.